database/events_schema.js

```diff
@@ -2,17 +2,17 @@ const mongoose = require('mongoose');
 
 const eventsSchema = mongoose.Schema({
   eventTitle: {type: String, required: true},
-  eventDate: {type: Date, min: Date.now(), required: true},
+  eventDate: {type: String, required: true},
```

Comment:
why you've changed date to string?

Reply:
oh sorry i just saw this didn't notice it before.
cause if i could remember right i had issues when trying to save data into the database, cause it would always fail validation.